diff --git a/node_modules/astro/content-types.template.d.ts b/node_modules/astro/content-types.template.d.ts
index 596764f..b6e4507 100644
--- a/node_modules/astro/content-types.template.d.ts
+++ b/node_modules/astro/content-types.template.d.ts
@@ -167,11 +167,11 @@ declare module 'astro:content' {
 		ReturnTypeOrOriginal<Required<ContentConfig['collections'][C]>['schema']>
 	>;
 
-	type ContentEntryMap = {
+	export type ContentEntryMap = {
 		// @@CONTENT_ENTRY_MAP@@
 	};
 
-	type DataEntryMap = {
+	export type DataEntryMap = {
 		// @@DATA_ENTRY_MAP@@
 	};
 
